spring-integration-file/src/main/resources/org/springframework/integration/file/config/spring-integration-file-5.0.xsd

```diff
@@ -828,6 +828,15 @@ Only files matching this regular expression will be picked up by this adapter.
 					]]></xsd:documentation>
 				</xsd:annotation>
 			</xsd:enumeration>
+			<xsd:enumeration value="REPLACE_IF_MODIFIED">
+				<xsd:annotation>
+					<xsd:documentation><![CDATA[
+						If the local file already exists, it will be overwritten only
+						if the last modified timestamp does not match the source
+						time stamp. Only applies to local files.
```

Comment:
Ah - yes - in future it would be clearer if you said `s/time stamp/timestamp/`, or explicitly "space between time and stamp" rather than a general statement 😄